Files/cart/utils.py

```diff
@@ -1,8 +1,27 @@
 from Files import db
 from flask import jsonify
+from Files import cors
 from ..models import Cart, CartSchema, Product, Consultation
 
+#implicitly used functions
+def is_Already_in_Cart(user_id, pro_con_id, type):
+    if type == "product":
+        pro_con_id="P"+str(pro_con_id)
+    elif type == "consultation":
+        pro_con_id="C"+str(pro_con_id)
+    cart = Cart.query.filter_by(customer_id=user_id, pro_con_id=pro_con_id, item_type='cart').first()
+    if cart:
+        return True
+    return False
+
+
+# route Functions
 def AddCart(user_id, item_id, type):
+    if is_Already_in_Cart(user_id, item_id, type):
+        response=jsonify({"message": "Item already in cart"})
```

Comment:
Instead of this message, just increase the quantity by 1. Would be better that way.

Reply:
Yeah that should be handled by front-end I felt. Because otherwise the back-end architecture becomes clumsy. So if the front-end receives this message they will invoke the increase route